quiz/web/src/components/FormikTest/Formik.tsx

```diff
@@ -0,0 +1,120 @@
+import { Formik, Form, useField } from "formik";
+import * as Yup from "yup";
+import styled from "styled-components";
+import "./styles.css";
+import "./styles-custom.css";
+
+const MyTextInput = ({ label, ...props }) => {
+    // useField() returns [formik.getFieldProps(), formik.getFieldMeta()]
+    // which we can spread on <input> and alse replace ErrorMessage entirely.
+    const [field, meta] = useField(props);
+    return (
+        <>
+            <label htmlFor={props.id || props.name}>{label}</label>
+            <input className="text-input" {...field} {...props} />
+            {meta.touched && meta.error ? <div className="error">{meta.error}</div> : null}
+        </>
+    );
+};
+
+const MyCheckbox = ({ children, ...props }) => {
+    const [field, meta] = useField({ ...props, type: "checkbox" });
+    return (
+        <>
+            <label className="checkbox">
+                <input {...field} {...props} type="checkbox" />
+                {children}
+            </label>
+            {meta.touched && meta.error ? <div className="error">{meta.error}</div> : null}
+        </>
+    );
+};
+
+// Styled components ....
+const StyledSelect = styled.select`
+    color: var(--blue);
+`;
+
+const StyledErrorMessage = styled.div`
```

Comment:
Nie jestem do końca przekonany co do styles components.
`text-sm text-red-600 w-100 mt-1 dark:text-red-300`
Jest dużo krótsze a robi to samo, befora bym musial sobie przypomniec jak się robi.

Reply:
I tailwind jest kompatybilny z MUI 😅

Reply:
Rozumiem, ja podchodzę elastycznie, natomiast to usłyszałem na ostatnim spotkaniu, dlatego nie powstrzymywałem się przed dodaniem tej paczki, żeby użyć jej do testów. 
Jak dla mnie kwestia do przegadania jak najbardziej

Reply:
Framework do CSS'a został zmieniony na TailwindCSS zgodnie z ostatnią rozmową techniczna, dot. wyboru frameworka do CSS. Zamykam.